benchmark/text-generation-inference/llama3-70b-inf2.48xlarge/.env

```diff
@@ -0,0 +1,9 @@
+MODEL_ID='NousResearch/Meta-Llama-3-70B-Instruct'
+HF_BATCH_SIZE=4
+HF_SEQUENCE_LENGTH=4096
```

Comment:
These two variables are not required anymore, since their value can be deduced from MAX_BATCH_SIZE and MAX_TOTAL_TOKENS.

Reply:
Removed from all the .env files and all the docker compose files (including llama-7b and mistral-7b examples).  Tested on the latest HF DLAMI with 0.0.23 TGI image.